index.html

```diff
@@ -0,0 +1,231 @@
+                                    Reminders
+                                </a>
+                            </li>
+
+                            <li>
+                                <a class="dropdown-item d-flex gap-3" href="#">
+                                    <img src="images/icon-planning.svg" alt="Toto list icon" width="17px" height="17px">
+                                    Planning
+                                </a>
+                            </li>
+                        </ul>
+                    </li>
+                    <li class="nav-item dropdown">
+                        <a class="nav-link nav-rotate-tow d-flex flex-direction-row align-items-center gap-2" href="#">
+                            Company
+                            <i class="fa-solid fa-angle-down fa-xs rotate-tow"></i>
+                        </a>
+
+                        <ul class="dropdown-menu dropdown-list-tow">
+                            <li>
+                                <a class="dropdown-item d-flex" href="#">History</a>
+                            </li>
+
+                            <li>
+                                <a class="dropdown-item d-flex" href="#">Our team</a>
+                            </li>
+
+                            <li>
+                                <a class="dropdown-item d-flex" href="#">Blog</a>
+                            </li>
+                        </ul>
+                    </li>
+
+                    <li class="nav-item">
+                        <a class="nav-link" href="#">Careers</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="#">About</a>
+                    </li>
+                </ul>
+
+                <ul class="navbar-nav">
+                    <li class="nav-item d-flex align-items-center">
+                        <a class="nav-link" href="#">Login</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link btn-mentor btn-mentor-white d-flex align-items-center" href="#">Register</a>
+                    </li>
+                </ul>
+            </div>
+
+            <div class="modal-backdrop fade"></div>
+        </nav>
+    </div>
+
+    <main>
+        <div class="row d-flex flex-column-reverse flex-md-row">
+            <div class="col-12 col-md-6 d-flex flex-column align-items-center justify-content-md-end ">
+                <h2 class="main-title d-flex justify-content-center flex-md-column">
+                    <span>Make</span> remote work
+                </h2>
+
+                <p class="main-desc d-flex justify-content-center">
+                    Get your team in sync, no matter your location. Streamline processes, create team rituals, and watch
+                    productivity soar.
+                </p>
+
+                <div class="d-flex justify-content-center w-100 justify-content-md-start">
```

Comment:
Using Flex space between rather than adding huge padding or margin for the elements